### PR TITLE
chore(docs): add Cam's information to openbeta-devs file

### PR DESCRIPTION
**Pull request by cammcinnes**
---
name: Pull request
about: Create a pull request
title: 'Add Cam to the contributor map'
labels: ''
assignees: '@actuallyyun'

---

## What type of PR is this?(check all applicable)
- [ ] refactor
- [ ] feature
- [ ] bug fix
- [x] documentation
- [ ] optimization
- [ ] other

### Related Issues

Issue #

### What this PR achieves

Dev Onboarding

### CODE REVIEW

File: src/assets/contributors-map.geojson [hunk 1]
```diff
@@ -180,6 +180,21 @@
...
+      "properties": {
+        "firstName": "Cam",
+        "githubId": "cammcinnes",
+        "favoriteCrag": "20a6f4b3-c767-53e9-a593-7b0d96484d4a"
+      },
+      "geometry": {
+        "type": "Point",
+        "coordinates": [
+          49.90644802932367, 
+          -123.16229248871166
```
Comment (vnugent) [inline src/assets/contributors-map.geojson]: Can you swap the coordinates?  In Geojson they are `[longitude, latitude]`

**Review by vnugent — CHANGES_REQUESTED** [review]

File: src/assets/contributors-map.geojson (hunk 1 above)
Comment (cammcinnes) [reply src/assets/contributors-map.geojson]: Yes, no problem! Done now

File: src/assets/contributors-map.geojson [hunk 2]
```diff
@@ -191,8 +191,8 @@
       "geometry": {
         "type": "Point",
         "coordinates": [
-          49.90644802932367, 
-          -123.16229248871166
+          -123.16229248871166,
+          49.90644802932367
```
Comment (vnugent) [inline src/assets/contributors-map.geojson:195]: @cammcinnes your pull request picked up other changes for some reason https://github.com/OpenBeta/open-tacos/pull/1144/files
The simplest way to fix this is abandon this PR and start a new one :)

**Review by vnugent — CHANGES_REQUESTED** [review]

**Comment by vnugent** [comment]
@cammcinnes please resubmit a new PR

**Pull request closed without merging**